vue/components/ui/molecules/upload-area/upload-area.vue

```diff
@@ -150,7 +150,14 @@ export const UploadArea = {
             return this.draggingIcon ? this.draggingIcon : "cloud-upload";
         },
         emptySlots() {
-            return Object.entries(this.$slots).length === 0 && !this.$scopedSlots?.default;
+            // forces this property to refresh on data change
+            const bindRefresh = Boolean(this.filesData) || !this.filesData;
+
+            return (
+                Object.entries(this.$slots).length === 0 &&
+                !this.$scopedSlots?.default &&
+                bindRefresh
+            );

```

Comment:
```suggestion
            return !this.noFileSelected && Object.entries(this.$slots).length === 0 && !this.$scopedSlots?.default;
```

use this instead and add this value to mixins/upload.js
```

    computed: {
        noFileSelected() {
            return this.filesData?.length === 0;
        }
    },
```

also, remove it from `input-file` component.

I think it looks more elegant. What do you say?